glass/cosmology.py

```diff
@@ -0,0 +1,25 @@
+"""Module for cosmology.api utilities."""
+
+from typing import Any, Protocol
+
+from numpy.typing import NDArray
+
+import cosmology.api
+
+Array = NDArray[Any]
+
+
+class Cosmology(
+    cosmology.api.HasComovingDistance[Array, Array],  # type: ignore[misc]
+    cosmology.api.HasCriticalDensity0[Array],  # type: ignore[misc]
+    cosmology.api.HasGrowthFactor[Array, Array],  # type: ignore[misc]
```

Comment:
I'm getting `module 'cosmology.api' has no attribute 'HasGrowthFactor'`. Any idea why? It appears to exist